test/env_file_test.rb

```diff
@@ -11,6 +11,16 @@ class EnvFileTest < ActiveSupport::TestCase
       Kamal::EnvFile.new(env).to_s
   end
 
+  test "to_s won't escape '#'" do
+    env = {
+      "foo" => "\#$foo",
```

Comment:
Can you change these to `#` instead of `\#`? They mean the same thing and the `\` makes it seem like there's some extra escaping going on that isn't really there.

Reply:
done!